src/app/form/(formSteps)/personal-details/2/PersonalDetailsStep2.test.tsx

```diff
@@ -8,20 +8,50 @@ import { render, screen, within } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import { type AppRouterInstance } from "next/dist/shared/lib/app-router-context.shared-runtime";
 
+const clickNoSameBillingMailingAddress = async () => {
```

Comment:
[dust] instead of `clickNoSameBillingMailingAddress` and `clickYesSameBillingMailingAddress`, maybe `clickSameBillingMailingAddressNo` and `clickSameBillingMailingAddressYes`? So only their suffixes are different?